src/CommonLib/aspnetcoreconfig.h

```diff
@@ -215,18 +215,38 @@ class ASPNETCORE_CONFIG : IHttpStoredContext
         return &m_struConfigPath;
     }
 
-    STRU*
-    QueryHostfxrPath()
+    PCWSTR*
+    QueryHostFxrArguments(
+        VOID
+    )
     {
-        return &m_struHostFxrPath;
+        return m_ppStrArguments;
     }
 
-    BOOL
-    QueryIsStandAloneApplication(
+    DWORD
+    QueryHostFxrArgCount(
+        VOID
+    )
+    {
+        return m_dwArgc;
+    }
+
+    STRU*
+    QueryHostFxrFullPath(
         VOID
     )
     {
-        return m_fIsStandAloneApplication;
+        return &m_struHostFxrLocation;
+    }
+
+    VOID
+    SetHostFxrArguments(
+        DWORD dwArgc,
+        PCWSTR* ppStrArguments
+    )
+    {
+        m_dwArgc = dwArgc;
```

Comment:
there will be memory leak if this function is called more than once. please free m_ppStrArgments first if it is not NULL and then do the assignment